### PR TITLE
chore(release): pulling release/2.0.0 into master

### PR DESCRIPTION
**Pull request by devops-github-rudderstack**
:crown: *An automated PR*

   1.0.0 (2022-12-28)<br> * feat!: update MoEngage SDK to the latest version 9.1.0 ([20aa6ec](https://github.com/rudderlabs/rudder-integration-moengage-ios/commit/20aa6ec))<br> * ci: fix publish-new-release ([3224228](https://github.com/rudderlabs/rudder-integration-moengage-ios/commit/3224228))

### CODE REVIEW

File: Rudder-Moengage/Classes/RudderMoengageIntegration.m [hunk 1]
```diff
@@ -66,83 +81,73 @@ - (void) processRudderEvent: (nonnull RSMessage *) message {
...
-        [traits removeObjectForKey:@"lastname"];
-      }
-        
-      if ([traits objectForKey:@"gender"]) {
-        [[MoEngage sharedInstance] setUserAttribute:[traits objectForKey:@"gender"] forKey:USER_ATTRIBUTE_USER_GENDER];
-        [traits removeObjectForKey:@"gender"];
+      
+      NSString *gender =[traits objectForKey:@"gender"];
+      if (gender) {
+        if ([gender  isEqual: @"M"] || [gender  isEqualToString: @"MALE"]) {
```
Comment (pallabmaiti) [inline Rudder-Moengage/Classes/RudderMoengageIntegration.m:124]: Can we use only 1 of them by our specs?

Comment (1abhishekpandey) [reply Rudder-Moengage/Classes/RudderMoengageIntegration.m:124]: We don't seems to have any specs defined w.r.t. Gender.
Only instance of gender reference, I could find in the Facebook Custom Audience doc, refer [here](https://www.rudderstack.com/docs/destinations/streaming-destinations/fb-custom-audience/#schema-fields-mapping:~:text=Use%20these%20values,for%20female.)

Comment (1abhishekpandey) [reply Rudder-Moengage/Classes/RudderMoengageIntegration.m:124]: Moreover, I tried to sync the gender code as per MoEngage Android implementation.

Comment (pallabmaiti) [reply Rudder-Moengage/Classes/RudderMoengageIntegration.m:124]: Cool.

**Review by pallabmaiti — APPROVED** [review]

**Review by desusai7 — APPROVED** [review]

**Pull request merged**